rtic-monotonics/src/stm32.rs

```diff
@@ -238,8 +238,8 @@ macro_rules! make_timer {
                 $timer.cr1().modify(|r| r.set_cen(false));
 
                 assert!((tim_clock_hz % timer_hz) == 0, "Unable to find suitable timer prescaler value!");
-                let psc = tim_clock_hz / timer_hz - 1;
-                $timer.psc().write(|r| r.set_psc(psc as u16));
+                let psc = u16::try_from(tim_clock_hz / timer_hz - 1).expect("Clock prescaler overflowed!");
```

Comment:
To not have formatting bloat unconditionally added.

```suggestion
                let Ok(psc) = u16::try_from(tim_clock_hz / timer_hz - 1) else {
                    panic!("Clock prescaler overflowed!");
                };
```

Reply:
thanks, updated